trakked-connector.php

```diff
@@ -57,7 +58,11 @@
         ]);
     }
 
+    $pluginData = get_file_data(__FILE__, array('Version' => 'Version'), false);
```

Comment:
Boolean attribute is not allowed for `$context` AFAIK.

```suggestion
    $pluginData = get_file_data(__FILE__, array('Version' => 'Version'));
```